File: Obj-C/Lab7-ListDemo/Lab7-ListDemo/CollectionViewDemoVC/myCollectionViewCell/myCollectionViewCell.m
//
//  myCollectionViewCell.m
//  Lab7-ListDemo
//
//  Created by 張力元 on 2019/3/7.
//  Copyright © 2019 張力元. All rights reserved.
//

#import "myCollectionViewCell.h"

@interface myCollectionViewCell()
@property (weak, nonatomic) IBOutlet UIImageView *imgView;
@property (weak, nonatomic) IBOutlet UILabel *titleLabel;
@end

@implementation myCollectionViewCell
- (void)awakeFromNib {
    [super awakeFromNib];
}

- (void)setCellImage:(NSString *)imageName title:(NSString *)title{
    [_imgView setImage:[UIImage imageNamed:imageName]];
    _titleLabel.text = title;
}
@end

File: Swift/lab7/lab7_exercise/lab7_exercise/collectionViewDemoVC/myCollectionCell/myCollectionViewCell.swift
//
//  myCollectionViewCell.swift
//  lab7-1-4
//
//  Created by taizhou on 2019/7/30.
//  Copyright © 2019 taizhou. All rights reserved.
//

import UIKit

class myCollectionViewCell: UICollectionViewCell {

    
    @IBOutlet var imgView: UIImageView!
    
    @IBOutlet var titleLabel: UILabel!
    
    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }

    func setCell(imgName:String,title:String){
        imgView.image = UIImage(named: imgName)
        titleLabel.text = title
    }
}
